package.json

```diff
@@ -43,7 +43,7 @@
     "http-errors": "^1.8.0",
     "lodash": "^4.17.15",
     "mime-types": "^2.1.26",
-    "serverless-http": "^2.3.1",
+    "serverless-http": "^3.2.0",
```

Comment:
Why not version 3.3.0 like in deployment

Reply:
Negligence. I can go through each repo and sync these 👍🏼

Reply:
The `serverless-http` of the deployment repo was also 3.2.0. https://github.com/snkeos/fhir-works-on-aws-deployment/blob/322e6f627640ce0df78461a1c08fa7cf2847af43/package.json#L47

While the `serverless` package is 3.3.0. https://github.com/snkeos/fhir-works-on-aws-deployment/blob/322e6f627640ce0df78461a1c08fa7cf2847af43/package.json#L75